engine/src/test/java/org/camunda/bpm/engine/test/logging/ProcessDataLoggingContextTest.java

```diff
@@ -164,6 +170,131 @@ public void shouldLogMdcPropertiesOnlyInActivityContext() {
     assertActivityLogsPresentWithoutMdc("ENGINE-130");
   }
 
+  @Test
+  @WatchLogger(loggerNames = {PVM_LOGGER, CMD_LOGGER}, level = "DEBUG")
+  public void shouldPreserveMDCExternalPropertiesAfterJobCompletion() {
+    // given
+
+    // a set of custom Logging Context parameters that populate the MDC prior to any process instance execution
+    testMDCFacade.withDefaultLoggingContextParameters(
+        "customActivityId",
+        "customActivityName",
+        "customApplicationName",
+        "customBusinessKey",
+        "customProcessDefinitionId",
+        "customProcessDefinitionKey",
+        "customProcessInstanceId",
+        "customTenantId",
+        "customEngineName"
+    );
```

Comment:
🔧 Could be moved to a method so you don't need to repeat it 3 times.

Reply:
❓ Do you think it would be easier for the reader to hide this call into a method? I thought the visibility of the logging context parameters configuration makes it more transparent for the reader to understand which parameters are used. If you vote yes, i will extract it into a method.

Reply:
It's just a test so I don't have a firm opinion, I just noticed that these 10 or so lines are repeated 3 times and found it unnecessary. Up to you!